compiler_opt/tools/extract_ir_lib.py

```diff
@@ -314,14 +350,20 @@ def run_extraction(objs: List[TrainingIRExtractor], num_workers: int,
       bitcode embedding.
     bitcode_section_name: The name of the bitcode section created by the
       bitcode embedding.
+    default_command_line_flags: The command line flags to use if no command
+      line is present from within the build (e.g., extracting from a directory
+      containing only bitcode).
+    input_type: The type of input that IR is being extracted from.
   """
   extract_artifacts = functools.partial(
       TrainingIRExtractor.extract,
       llvm_objcopy_path=llvm_objcopy_path,
       cmd_filter=cmd_filter,
       thinlto_build=thinlto_build,
       cmd_section_name=cmd_section_name,
-      bitcode_section_name=bitcode_section_name)
+      bitcode_section_name=bitcode_section_name,
+      default_command_line_flags=default_command_line_flags,
```

Comment:
can this notion of "default_command_line_flags" be joined with `global_command_override` below?